openslides_backend/action/actions/motion_state/create_update_delete.py

```diff
@@ -62,6 +62,7 @@ class MotionStateActionSet(ActionSet):
             "merge_amendment_into_final",
             "show_recommendation_extension_field",
             "first_state_of_workflow_id",
+            "next_state_ids",
```

Comment:
This seems fine for me, please document this in the wiki (https://github.com/OpenSlides/OpenSlides/wiki/motion_state.create) and add a test for it.

Reply:
removed. Field belongs to meeting.create
The field "first_state_of_workflow_id" is since 10/2020 in the crate schema present. wiki updated [motion_state.create Payload](https://github.com/OpenSlides/OpenSlides/wiki/motion_state.create)